### PR TITLE
Minor formatting and addition of !feedback command

### PR DESCRIPTION
**Pull request by JoelLucaAdams**

### CODE REVIEW

File: demobot.py [hunk 1]
```diff
@@ -6,8 +6,23 @@
...
+# logs data to the discord.log file, if this file doesn't exist at runtime it is created automatically
+logger = logging.getLogger('discord')
+logger.setLevel(logging.INFO)  # logging levels: NOTSET (all), DEBUG (bot interactions), INFO (bot connected etc)
+handler = logging.FileHandler(filename='discord.log', encoding='utf-8', mode='w')
+handler.setFormatter(logging.Formatter('%(asctime)s:%(levelname)s:%(name)s: %(message)s'))
+logger.addHandler(handler)
+
+
+# Initialise the Bot object with an accessible help Command object
+helpCommand = DefaultHelpCommand()
```
Comment (IdrisTheDragon) [inline demobot.py:18]: why move this?

**Comment by JoelLucaAdams** [comment]
Not a clue on how Pipfile.lock got there tbh

File: demobot.py [hunk 2]
```diff
@@ -22,34 +37,35 @@ class General(commands.Cog):
...
+    @commands.command()
+    async def feedback(self, ctx: Context):
+        """
+        Report feedback or issues with the bot
+        """
+        await ctx.send('If the bot is broken or you have any feedback you\'d like to submit please join '
+                       'https://discord.gg/b3EdxVK and post a message in the <#740966780079571105> or '
+                       '<#740967688876327012> channels')
 
-# Initialise the Bot object with an accesible help Command object
```
Comment (IdrisTheDragon) [inline demobot.py:39]: This setup stuff is deliberately kept together with the general cog setup at the end to be easier to read, without the definition of the General cogs commands stuck between. Ideally General cog should be moved to a separate file at this point due to it's size.

File: Pipfile [hunk 3]
```diff
@@ -8,6 +8,7 @@ verify_ssl = true
 [packages]
 discord = "*"
 python-dotenv = "*"
+PyNaCl = "*"
```
Comment (IdrisTheDragon) [inline Pipfile:11]: what's this new dependency for? doesn't seem to be used anywhere?

File: demobot.py (hunk 1 above)
Comment (JoelLucaAdams) [reply demobot.py:18]: Preferred the new order of the code as it makes more sense to me

**Comment by IdrisTheDragon** [comment]
> Not a clue on how Pipfile.lock got there tbh

because it was already being tracked so the gitignore doesn't pick up on it. Now it is deleted from git the .gitignore should ignore local changes to the lock file

File: Pipfile (hunk 3 above)
Comment (JoelLucaAdams) [reply Pipfile:11]: Discord complains about it not existing for voice connecting reasons so I've added it for when we add voice connectability in the future and to remove the warning message

File: demobot.py (hunk 2 above)
Comment (JoelLucaAdams) [reply demobot.py:39]: Once this branch is merged I'll move all the general cog stuff to a seperate file

**Review by IdrisTheDragon — APPROVED** [review]

**Pull request merged**